.github/scripts/determine-runners.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require "formula"
+
+formula = Formula[ARGV[0]]
+timeout = ARGV[1].to_i
```

Comment:
Would be nice to handle if there's <0, >2 args here or just use our proper argument parsing.